src/components/eventSingleComponent/image.stories.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+import { html } from 'lit';
+import './image';
+
+export default {
+  title: 'Components/Image',
+  component: `kd-image`,
+  argTypes: {
+    mediaPosition: {
+      control: { type: 'radio' },
+      options: ['left', 'right'],
+      description: 'Media on the left or right',
+    },
+
+    bleedMedia: {
+      control: { type: 'radio' },
+      options: ['true', 'false'],
+      description: 'Bleed the media slot to the edge of the browser',
+    },
+    headlineText: {
+      control: { type: 'text' },
+      description: 'Headline Text (required)',
+    },
+    eyebrowText: {
+      control: { type: 'text' },
+      description: 'Eyebrow Text',
+    },
+    buttonLabel: {
+      control: { type: 'text' },
+      description: 'Button Label',
+    },
+    buttonLink: {
+      control: { type: 'text' },
+      description: 'Button Link',
+    },
+  },
+};
+
+const args = {
+  mediaPosition: 'left',
+  bleedMedia: 'false',
+  date: '12:00 PM EST',
+  location: 'Sydney',
+  headlineText: 'Lorem ipsum dolor sit amet dsdsdsdsdssd',
+  //   buttonLabel: 'Button text',
+  buttonLink: 'https://www.kyndryl.com',
+  eventSingleBaseComponent: 'yes',
+};
+
+export const alternatingFeatureWithImage = {
```

Comment:
Change this name. It makes storybook add this under the Image component.
![image](https://github.com/kyndryl-design-system/shidoka-content/assets/1440548/bfb93c7e-d05f-42f1-90ef-8b981faacbfe)